.ruby-gemset

```diff
@@ -0,0 +1 @@
+yesware
```

Comment:
It'd be nice to update our CI system to not require the gemset file, since it's not used anymore in lieu of Bundler 😬 but fine for now!

Reply:
Ah, good to know! I'll keep that in mind for the future. Hopefully we can retire use of this old gem soon!